### PR TITLE
Dented supernova - handle sorting of undeleted projects + sorting of user projects by updatedAt on profile pages

### PR DESCRIPTION
**Pull request by ttseng**
https://dented-supernova.glitch.me/

Currently, when the user undeletes a project, the undeleted project is shown as their most recent project.  However, upon refreshing the page, the undeleted project is no longer seen at the top of the list as user projects on the profile page are not automatically sorted by when they are last updated.

To test:
* open https://dented-supernova.glitch.me/ and try undeleting one of your projects.  Then refresh the page and check that the undeleted project still appears at the top of the list.

Manuscript case: https://glitch.manuscript.com/f/cases/3327976

### CODE REVIEW

File: src/presenters/user-editor.js [hunk 1]
```diff
@@ -124,6 +124,8 @@ class UserEditor extends React.Component {
         console.warn('failed to rename project on undelete', e);
       }
     }
+    // temp set undeleted project updatedAt to now, while it's actually updating behind the scenes
+    data.updatedAt = Date.now();
```
Comment (clottman) [inline src/presenters/user-editor.js]: This will return the date in seconds since the unix epoch - one of my projects' updatedAt is formatted like: 
```
"updatedAt":"2019-03-04T21:48:33.585Z"
```
I think you probably want `new Date()` ?

Comment (clottman) [reply src/presenters/user-editor.js]: Also is this change necessary? We're doing the sort in ProjectsLoader and not updating ProjectsLoader.projects, so I'm not sure what component would be checking updatedAt on the newly added project.

Comment (ttseng) [reply src/presenters/user-editor.js]: I think it's necessary to update the undeleted projects `updatedAt` because the ProjectsLoader in `user.js` is given the user's recent projects , now ordered by updatedAt date.  

If we don't update the date, `data` from `undeleteProject` uses the `updatedAt` date from the project at the time it was deleted, so the project doesn't appear at the top of the list.

Comment (clottman) [reply src/presenters/user-editor.js]: It's really hard to follow the data flow between the two components, but it's working for me in ~dashing-caribou without this line.

Comment (ttseng) [reply src/presenters/user-editor.js]: hm, it's not working for me. did you try undeleting something you deleted a long time ago?

Comment (clottman) [reply src/presenters/user-editor.js]: Yup

Comment (clottman) [reply src/presenters/user-editor.js]: Hmm - now it's not working for me either. Maybe I was testing on an old build. Never mind then!

**Comment by ttseng** [comment]
Added the space to the pop-over title for adding projects to a collection: https://glitch.manuscript.com/f/cases/3327962/
image
